addons/website_sale_slides/__manifest__.py

```diff
@@ -9,7 +9,6 @@
 
     'depends': ['website_slides', 'website_sale'],
     'installable': True,
-    'auto_install': True,
```

Comment:
Is it intended ?

Reply:
I deleted this to test when we unchecked the option to sell courses if the buttons weren't present anymore. But when I try in the first place, I couldn't do it because each time I unchecked the option it was automatically reinstall just after.